src/System.Private.Xml/tests/XmlReaderLib/CommonTest.cs

```diff
@@ -6,7 +6,7 @@
 using System.IO;
 using XmlCoreTest.Common;
 
-[assembly: System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
+[assembly: System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAssembly]
 
 namespace System.Xml.Tests
```

Comment:
I think this should be excluded anyway since `-[*.Tests]*` is also used on the OpenCover call, but maybe there's other namespaces in the assembly and certainly some of the other cases in this PR do need the attribute. Having more test assemblies excluded by the  `-[*.Tests]*` rule than by attribute might be a sensible longer-term goal though.